src/js/nav-secondary.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+/*global document window*/
+
+(function() {
+  const navigation = document.querySelector('[data-nav-secondary');
+
+  if (navigation) {
+    const toggle = navigation.querySelector('[data-nav-secondary-toggle]');
+    const navigation_sub = navigation.querySelector('[data-nav-secondary-sub]');
+    const media_query = window.matchMedia('(width < 64em)')
+  
+    navigation.classList.add('js-nav-secondary');
+  
+    function toggle_nav(e) {
+      if (navigation.classList.contains("js-nav-secondary--opened")) {
+        navigation.classList.remove('js-nav-secondary--opened');
```

Comment:
Wonder if we might be able to use some `aria-pressed`/`aria-expanded` semantics here to manage button state. E.g. .https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#associated_aria_roles_states_and_properties

Might also mean we add/remove `role="button"` from control at different breakpoints?

Reply:
Yeah, nice. I've switched over to `aria-pressed/aria-expanded` and added the `role="button"` when needed. Just needed a minor adjustment to the CSS to pull it all together.